### PR TITLE
[NestJS] Global prefix support

### PR DESCRIPTION
**Pull request by DennisSnijder**
## Context
See #591

## What has been done
- Checked for the existence of a global prefix in the ApplicationConfig from NestJS

### CODE REVIEW

**Comment by DennisSnijder** [comment]
@felixmosh ready for review 😋

**Comment by felixmosh** [comment]
closes #591

File: packages/nestjs/src/bull-board.root-module.ts [hunk 1]
```diff
@@ -2,21 +2,27 @@ import { DynamicModule, Inject, MiddlewareConsumer, Module, NestModule, Provider
...
     private readonly adapterHost: HttpAdapterHost,
+    private readonly applicationConfig: ApplicationConfig,
     @Inject(BULL_BOARD_ADAPTER) private readonly adapter: BullBoardServerAdapter,
     @Inject(BULL_BOARD_OPTIONS) private readonly options: BullBoardModuleOptions
   ) {
   }
 
   configure(consumer: MiddlewareConsumer): any {
-    this.adapter.setBasePath(this.options.route);
+
```
Comment (felixmosh) [inline packages/nestjs/src/bull-board.root-module.ts:20]: I prefer it to be something like this:
```js
const globalPrefix = this.applicationConfig.getGlobalPrefix() || '';
this.adapter.setBasePath(`${ globalPrefix }${ this.options.route }`);
```

Comment (DennisSnijder) [reply packages/nestjs/src/bull-board.root-module.ts:20]: Good one!

**Comment by felixmosh** [comment]
closes #591

File: packages/nestjs/src/bull-board.root-module.ts (hunk 1 above)
Comment (DennisSnijder) [reply packages/nestjs/src/bull-board.root-module.ts]: Done!

**Comment by felixmosh** [comment]
Thank you @DennisSnijder & @SimonRosenau 🙏🏼
Released in `v5.5.2`

**Comment by piotr-skowronski-prpl** [comment]
@DennisSnijder It looks like there's still a bug here.
When setting the global prefix without the `/` at the beginning, like so `app.setGlobalPrefix('api');`, assets are taken from `/api/api/queues`.
And the official NestJS documentation does not use the leading `/` in the global prefix section so people will be tripping over this.

**Comment by DennisSnijder** [comment]
> @DennisSnijder It looks like there's still a bug here. When setting the global prefix without the `/` at the beginning, like so `app.setGlobalPrefix('api');`, assets are taken from `/api/api/queues` which doesn't exist, so it fails. And the official NestJS documentation does not use the leading `/` in the global prefix section so people will be tripping over this.

Good catch! i'll take a look into it later today!

**Pull request merged**